docs/src/starknet/declare-deploy.md

```diff
@@ -0,0 +1,33 @@
+# Joining the Declaration and Deployment Together
+
+`sncast` allows declaring and deploying contracts through dedicated commands. However, sometimes contracts are meant to be only instantiated once, thus the standard way of declaration and deployment is unnecessarily time-consuming.
```

Comment:
I'd just say it's an simpler solution when you want to immediately deploy a declared contract.